openc3/python/openc3/script/__init__.py

```diff
@@ -47,7 +47,7 @@ def disconnect_script():
 ###########################################################################
 
 from .api_shared import *
-from .calendar import *
+from .cosmos_calendar import *
```

Comment:
Had to change this name because Python was importing our calendar instead of the system calendar